### PR TITLE
Add Balatro + Love2D support via lovely-injector

### PR DESCRIPTION
**Pull request by ethangreen-dev**
_The poker roguelike. Balatro is a hypnotically satisfying deckbuilder where you play illegal poker hands, discover game-changing jokers, and trigger adrenaline-pumping, outrageous combos._

https://store.steampowered.com/app/2379780/Balatro/

lovely-injector is my "JIT-like" lua injector for Love2D games: https://github.com/ethangreen-dev/lovely-injector. It injects lua into the game process at runtime, bypassing the need for manual executable patching.

### CODE REVIEW

**Review by Oksamies — APPROVED** [review]
Works

File: src/model/game/GameManager.ts [hunk 1]
```diff
@@ -594,6 +595,12 @@ export default class GameManager {
             "https://thunderstore.io/c/content-warning/api/v1/package/", EXCLUSIONS,
             [new StorePlatformMetadata(StorePlatform.STEAM, "2881650")], "ContentWarning.png",
             GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.BEPINEX, []),
+
+        new Game("Balatro", "Balatro", "Balatro",
+            "Balatro", ["Balatro.exe"], "Balatro_Data",
+            "https://thunderstore.io/c/balatro/api/v1/package/", EXCLUSIONS,
+            [new StorePlatformMetadata(StorePlatform.STEAM, "2379780")], "Balatro.png",
+            GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.LOVELY, []),
```
Comment (anttimaki) [inline src/model/game/GameManager.ts:603]: #1287 adds missing shorthands (the empty array at the end) for other entries here. IDK what they're used for or how one derives a shorthand for a game, but maybe Balatro should have one too?

**Review by anttimaki — APPROVED** [review]
Aside from the one comment LGTM, although I just reviewed what's in the PR and feel I have no comprehension of the possible implications regarding rest of the code base.

File: src/model/game/GameManager.ts (hunk 1 above)
Comment (Oksamies) [reply src/model/game/GameManager.ts:603]: Mythic has said that these shorthands should be ignored and left empty 🤷 Didn't get the reason though

Comment (ebkr) [reply src/model/game/GameManager.ts:603]: They're just shorthands for searching on the game selection screen

**Pull request merged**